department-of-veterans-affairs/vets-website
# Pension 69960 mvp current reason for separation

## Summary
On the Pension Benefits application(21P-527EZ), update the 'Health and employment information' chapter

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Start or continue the application
3. Go to the 'Household Information' chapter
4. Select 'Separated' on the 'Marital Status' page
6. Continue and complete the 'Current spouse's name' page
7. Continue and complete the 'Current marriage information' page
8. Continue and verify the 'Reason for separation' page is next
9. Verify the content and design of the page
10. Select 'Other' and verify that a required text field is displayed
11. Verify that all changes are saved to the redux store in form data and included in the save form api call

## Related issue(s)
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69960)
[Epic in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done

## Screenshots

### Desktop
| Reason for separation | Reason for separation (Required) | Reason for separation (Other) |
| ------ | ------ | ------ | 
|![localhost_3001_pension_application_527EZ_ (1)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/464f9f97-12ea-4ef6-9270-8fc79aca2a65)|![localhost_3001_pension_application_527EZ_ (2)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/62d14f2a-b2e1-4be4-8626-4ef1212642ea)|![localhost_3001_pension_application_527EZ_ (3)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/77644125-5be5-494b-b6ab-31e5c1945233)


## What areas of the site does it impact?
Pension Benefits Application

## Acceptance criteria
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69960)

### Quality Assurance & Testing

### Error Handling


### Authentication
